### PR TITLE
Docker: getting container IPs of container with network(s)

### PR DESCRIPTION
**Pull request by aledsage**

### CODE REVIEW

File: docker/src/main/java/org/jclouds/docker/domain/NetworkSettings.java [hunk 1]
```diff
@@ -59,6 +59,77 @@ public static Details create(String endpointId, String gateway, String ipAddress
          return new AutoValue_NetworkSettings_Details(endpointId, gateway, ipAddress, ipPrefixLen, ipv6Gateway, globalIPv6Address, globalIPv6PrefixLen,
                  macAddress);
       }
+      
+      public static Builder builder() {
```
Comment (andreaturli) [inline docker/src/main/java/org/jclouds/docker/domain/NetworkSettings.java]: as you are adding the builders, can you please use [this](https://github.com/google/auto/blob/master/value/userguide/builders.md) ?

**Comment by andreaturli** [comment]
minor comment on the builders. thanks @aledsage

**Comment by grkvlt** [comment]
Looks good @aledsage, just need a version for 1.9.x as well

**Comment by aledsage** [comment]
Thanks @andreaturli for pointing out the `@AutoValue.Builder` usage - so much nicer!
We should re-write the other builders (in master, rather than 1.9.x branch I presume) with that pattern as well. But in a separate PR obviously.

**Comment by grkvlt** [comment]
@aledsage yes, it's much nicer, and agree should update master docker DTOs like this

**Comment by andreaturli** [comment]
thanks @aledsage, merging now!

**Comment by andreaturli** [comment]
Merged at [master](http://git-wip-us.apache.org/repos/asf/jclouds-labs/commit/6dcadf1f)

**Pull request closed without merging**